tests/reorder_python_imports_test.py

```diff
@@ -885,6 +885,20 @@ def test_py3_plus_does_not_unsix_moves_urllib(tmpdir):
     assert f.read() == 'from six.moves import urllib\n'
 
 
+def test_py3_plus_renames_mock(tmpdir):
+    f = tmpdir.join('f.py')
+    f.write('from mock import patch\n')
+    assert main((str(f), '--py3-plus'))
+    assert f.read() == 'from unittest.mock import patch\n'
+
+
+def test_py3_plus_does_not_rename_mock(tmpdir):
```

Comment:
not_py3_plus maybe?

Reply:
reworked these and added a py33-plus option